homeassistant/components/overkiz/const.py

```diff
@@ -97,3 +116,42 @@
     OverkizCommandParam.SFC: "sfc",
     OverkizCommandParam.UPS: "ups",
 }
+
+OVERKIZ_UNIT_TO_HA: dict[str, str] = {
+    MeasuredValueType.ABSOLUTE_VALUE: "",
+    MeasuredValueType.ANGLE_IN_DEGREES: DEGREE,
+    MeasuredValueType.ANGULAR_SPEED_IN_DEGREES_PER_SECOND: f"{DEGREE}/{UnitOfTime.SECONDS}",
+    MeasuredValueType.ELECTRICAL_ENERGY_IN_KWH: UnitOfEnergy.KILO_WATT_HOUR,
+    MeasuredValueType.ELECTRICAL_ENERGY_IN_WH: UnitOfEnergy.WATT_HOUR,
+    MeasuredValueType.ELECTRICAL_POWER_IN_KW: UnitOfPower.KILO_WATT,
+    MeasuredValueType.ELECTRICAL_POWER_IN_W: UnitOfPower.WATT,
+    MeasuredValueType.ELECTRIC_CURRENT_IN_AMPERE: UnitOfElectricCurrent.AMPERE,
+    MeasuredValueType.ELECTRIC_CURRENT_IN_MILLI_AMPERE: UnitOfElectricCurrent.MILLIAMPERE,
+    MeasuredValueType.ENERGY_IN_CAL: "cal",
+    MeasuredValueType.ENERGY_IN_KCAL: "kcal",
+    MeasuredValueType.FLOW_IN_LITRE_PER_SECOND: f"{UnitOfVolume.LITERS}/{UnitOfTime.SECONDS}",
+    MeasuredValueType.FLOW_IN_METER_CUBE_PER_HOUR: UnitOfVolumeFlowRate.CUBIC_METERS_PER_HOUR,
+    MeasuredValueType.FLOW_IN_METER_CUBE_PER_SECOND: f"{UnitOfVolume.CUBIC_METERS}/{UnitOfTime.SECONDS}",
+    MeasuredValueType.FOSSIL_ENERGY_IN_WH: UnitOfEnergy.WATT_HOUR,
+    MeasuredValueType.GRADIENT_IN_PERCENTAGE_PER_SECOND: f"{PERCENTAGE}/{UnitOfTime.SECONDS}",
+    MeasuredValueType.LENGTH_IN_METER: UnitOfLength.METERS,
+    MeasuredValueType.LINEAR_SPEED_IN_METER_PER_SECOND: UnitOfSpeed.METERS_PER_SECOND,
+    MeasuredValueType.LUMINANCE_IN_LUX: LIGHT_LUX,
+    MeasuredValueType.PARTS_PER_BILLION: CONCENTRATION_PARTS_PER_BILLION,
+    MeasuredValueType.PARTS_PER_MILLION: CONCENTRATION_PARTS_PER_MILLION,
+    MeasuredValueType.PARTS_PER_QUADRILLION: "ppq",
+    MeasuredValueType.PARTS_PER_TRILLION: "ppt",
+    MeasuredValueType.POWER_PER_SQUARE_METER: UnitOfIrradiance.WATTS_PER_SQUARE_METER,
+    MeasuredValueType.PRESSURE_IN_HPA: UnitOfPressure.HPA,
+    MeasuredValueType.PRESSURE_IN_MILLI_BAR: UnitOfPressure.MBAR,
+    MeasuredValueType.RELATIVE_VALUE_IN_PERCENTAGE: PERCENTAGE,
+    MeasuredValueType.TEMPERATURE_IN_CELCIUS: UnitOfTemperature.CELSIUS,
+    MeasuredValueType.TEMPERATURE_IN_KELVIN: UnitOfTemperature.KELVIN,
+    MeasuredValueType.TIME_IN_SECOND: UnitOfTime.SECONDS,
+    # MeasuredValueType.VECTOR_COORDINATE: "",
```

Comment:
What is this one? Perhaps just remove it if we don't use it?

Reply:
I prefer to keep it just in case on day we face it. But if you insist and core owner too, I will remove it :)